addons/zeusHC/functions/fnc_getSpawnMachineId.sqf

```diff
@@ -1,6 +1,8 @@
 #include "script_component.hpp"
 TRACE_1("Params",_this);
-params [["_allHCs", entities "HeadlessClient_F"]];
```

Comment:
This will remove some cycle saving I did in the transfer to HC stuff.
https://github.com/BourbonWarfare/POTATO/blob/parachuteDismounts/addons/zeusHC/functions/fnc_transferGroupsToHC.sqf#L24
So it doesn't look for new entities on every new call within it's run.